src/main/java/codeu/bot/BotActions.java

```diff
@@ -117,17 +117,17 @@ public void doAction(Object ... argsObjects) {
                 List<Message> userMessages = messageStore.getMessagesByAuthor(user.getId());
                 String content = "";
                 if (getMessageCount) {
-                    content += "Amount of messages: " + userMessages.size() + "\n";
+                    content += "<p>Amount of messages: " + userMessages.size() + "</p>";
```

Comment:
Instead of surrounding by <p> </p> we can instead add <br /> to the end of each line. So it would like like...

`content += "Amount of messages: " + userMessages.size() + "<br />"`
instead of 
`content += "<p>Amount of messages: " + userMessages.size() + "</p>"`

This will avoid having a weird amount of space between each line.

Reply:
As we talked about in the meeting. Maybe use bullet points instead. <ul>, <li> tags. Really whatever looks best would be good.